fume hood table updates

JasonZ08 opened the pull request:
![Create_Modal](https://github.com/Sustainabli/sustainabli/assets/87089616/e0906ded-1da5-4ef9-a6a6-582220b11c73)
![Edit_Modal](https://github.com/Sustainabli/sustainabli/assets/87089616/0a43e04c-1511-417e-8a73-5552d8aed6bd)
![Sash_Height](https://github.com/Sustainabli/sustainabli/assets/87089616/0edc316a-0a02-4c29-9df2-45c41403987b)

michaelli00 commented:
Nice job. Couple of comments

- Could you style the button to match the figma color (light blue). Create a constant in Constants.js for the hex of this color so we can reuse it later
- User should be able to update the fume hood name of an existing fume hood, so the fume hood field should be a textedit where the current value is the current fume hood name
- User shouldn't be able to edit the sensor MAC address. We use this as the id of the sensor

JasonZ08 commented:
![image](https://github.com/Sustainabli/sustainabli/assets/87089616/c19093ea-8208-41be-bd5e-ae4284ce8287)
![image](https://github.com/Sustainabli/sustainabli/assets/87089616/2cf01415-4da6-473a-9b62-3ac358b74823)
![image](https://github.com/Sustainabli/sustainabli/assets/87089616/c8543e84-1939-4f53-9dcf-1f1748052a8b)

Made button the same color as the progress bar. Do you want both of them to be the color on the figma or just the button to be the color on the figma? Added comments and also changed the modal forms. Let me know if theres anything else you want changed.

michaelli00 reviewed (commented):
Sorry first set of comments didn't go through for some reason. But otherwise looks good

michaelli00 commented on src/components/FumeHoodsPage/FumeHoodsPage.js:
```diff
@@ -7,174 +7,152 @@ import Row from 'react-bootstrap/Row';
-	arr.forEach((ele) => {
-		if (ele.fumeHood === sensorName) {
-			res = <ProgressBar variant='info' now={ele.normalizedCfm}/>
+		let res = arr.find((ele) => ele.fumeHood === sensorName)
```
We can guarantee the element will be found in the list. So we can simplify this to
```
const res = arr.find(ele => ele.fumeHood === sensorName);
return <ProgressBar variant='info' now={res.normalizedCfm}/>
```

michaelli00 commented on src/components/FumeHoodsPage/components/FumeModalForm.js:
```diff
@@ -0,0 +1,178 @@
+                <Form.Control
+                  type='input'
+                  placeholder='Lab'
+                  defaultValue='Lab'
```
Same thing as before. I don't think we need a defaultValue? Let me know if it's actually needed

michaelli00 commented on src/components/FumeHoodsPage/components/FumeModalForm.js:
```diff
@@ -0,0 +1,178 @@
+              </Form.Group>
+            </Modal.Body>
+            <Modal.Footer>
+              <Button variant='dark' type='submit'> Update Fume Hood </Button>
```
Did you update the color of the button to match the figma? Light blue?  Create a .scss file and add the styling there

michaelli00 reviewed (commented):
When we're done with review, can you squash the commits so that it's a single commit?
